ols-6.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: page
-title: Welcome to the 6th cohort of OLS program!
+title: Welcome to the 6th cohort of Open Life Science program!
```

Comment:
Please keep OLS there and in other pages

Reply:
I had changed them all to Open Life Science, because some had that instead of OLS.Will change all to OLS for consistency. Thank you @bebatut

Reply:
I have made a second commit on this @bebatut